src/persistence.js

```diff
@@ -356,7 +356,7 @@ export default function _Persistence(mpInstance) {
         }
 
         if (!mpInstance._Store.SDKConfig.useCookieStorage) {
-            localStorageData.l = mpInstance._Store.isLoggedIn ? 1 : 0;
+            localStorageData.l = mpInstance._Store.isLoggedIn; // ? 1 : 0;
```

Comment:
I don't think this should be changed.  We want to set it as a number in the `IPersistenceMinified`, but as a boolean on the store.

Reply:
here is a screenshot of a test app
![image](https://github.com/mParticle/mparticle-web-sdk/assets/5377436/bf47e8eb-0a66-4137-99cf-854b7c299943)